### ISSUE TITLE
saved_model_cli can't convert a severable model for Tensorflow:serving

### ISSUE THREAD

**Issue by Eloring** [tensorflow/tensorflow#29732]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): NO
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux RHEL 7.6
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below):1.13.1
- Python version: Python 2.7.16 :: Anaconda, Inc.
- CUDA/cuDNN version: CUDA 10.1, cuDNN 7.6
- GPU model and memory: Tesla K80, 12G

**Describe the current behavior**

Use the tool saved_model_cli to covert a severable model with TF-TRT:
~~~
saved_model_cli convert \
--dir "/path/to/mask_rcnn/saved_model" \
--output_dir "/path/to/trt-mask-rcnn" \
--tag_set serve \
tensorrt --precision_mode FP32 --max_batch_size 32 --is_dynamic_op True
~~~

## While I got a frozen_graph model and the variables fold is empty, which is not severable!
~~~
variables/
    NULL
saved_model.pb
~~~

**Describe the expected behavior**

The following structure is expected. https://www.tensorflow.org/guide/saved_model#structure_of_a_savedmodel_directory

~~~
assets/
assets.extra/
variables/
    variables.data-?????-of-?????
    variables.index
saved_model.pb|saved_model.pbtxt
~~~

My final target is to [Optimizing TensorFlow Serving performance with NVIDIA TensorRT](https://medium.com/tensorflow/optimizing-tensorflow-serving-performance-with-nvidia-tensorrt-6d8a2347869a), and the model that I want to launch is a pre-trained mask-rcnn downloaded from [tensorflow_model_zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md)

Any help would be grateful!

**Comment by Eloring** [comment]
I solve this by modify the models/research/object_detection/exporter.py to make it export a saved model.

**Comment by gadagashwini** [comment]
@Eloring Looks like you are able to save model and create .pb file but having problem with tensorflow serving. For better and faster support you can post this on [Tensorflow serving repo](https://github.com/tensorflow/serving/issues). Let us know if you are stuck. Thanks!

**Comment by Eloring** [comment]
@gadagashwini Thanks for your reply! I have solved the problem.
Although the variables fold is empty, the converted model can be launched on tensorflow:serving.

The procedures are:
1. prepare a pre-trained model (.ckpt and .config files)
2. export to a saved_model (.pb and variables), which can be launched on tf:serving
3. convert to another saved_model with TensorFlow-TensorRT (only .pb)

My questions is,  in the 2nd step, only saved_model.pb can not be launched, but in the 3rd step, it can.
Why?

**Comment by jvishnuvardhan** [comment]
@Eloring This is better suited for TF_serving repo. Please post it [here](https://github.com/tensorflow/serving/issues). After posting there, please close the issue here. Thanks!

**Comment by Eloring** [comment]
ok, thank you so much!
